src/QuadForm/Herm/Lattices.jl

```diff
@@ -750,3 +750,114 @@ function maximal_integral_lattice(V::HermSpace)
   return maximal_integral_lattice(L)
 end
 
+################################################################################
+#
+#  Intersection
+#
+################################################################################
+
+#=
+    _reconstruction_herm_lattice(L::ZLat, E::Hecke.NfRel{nf_elem}, 
+    f::Hecke.VecSpaceRes{Hecke.NfRel{nf_elem}, Hecke.NfRelElem{nf_elem}}, Vas)
+                  -> HermLat{} 
+    
+Takes the $\mathbb{Z}$-lattice $L$ and reconstructs a herm lattice with base field
+$E$ and ambient space $Vas$. 
+=#
+function _reconstruction_herm_lattice(L::ZLat, E::Hecke.NfRel{nf_elem}, f::Hecke.VecSpaceRes{Hecke.NfRel{nf_elem}, Hecke.NfRelElem{nf_elem}}, Vas)
+  M = basis_matrix(L)
+  OE = maximal_order(E)
+  n = absolute_degree(E)
+
+  K = base_field(E)
+  OK = maximal_order(K)
+  deg = div(ncols(M),n)
```

Comment:
```suggestion
  @assert divides(ncols(M), n)[1]
  deg = divexact(ncols(M),n)
```
you want to make sure that `n` divides the number of columns of `M`. Even if in practice it will be always the case in your code, this does not cost much and it is nice for further debugging.